ext/imap/php_imap.c

```diff
@@ -3061,10 +3061,16 @@ PHP_FUNCTION(imap_fetch_overview)
 
 	array_init(return_value);
 	for (unsigned long  i = 1; i <= imap_conn_struct->imap_stream->nmsgs; i++) {
-		MESSAGECACHE *elt = mail_elt(imap_conn_struct->imap_stream, i);
-		ENVELOPE *env = mail_fetch_structure(imap_conn_struct->imap_stream, i, NIL, NIL);
+		MESSAGECACHE *elt;
+		ENVELOPE *env;
 
-		if (!elt->sequence || !env) {
+		elt = mail_elt(imap_conn_struct->imap_stream, i);
```

Comment:
Might as well merge the declaration and assignment into a single line (same as below).
But honestly, this doesn't really matter so I'd say do what you're comfortable with. It's just my impression that new code follows the C99 style.

Reply:
I think I'll leave this one like this, as it makes it clear that we are working with two cclient structs in the loop